lib/core/src/lib/comments/comments.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="adf-comments-container">
-    <h1 id="comment-header" class="adf-comments-header">
+    <h2 id="comment-header" class="adf-comments-header">
```

Comment:
please use a regular div and add the required styles to the `adf-comments-header` if there's anything needed for the font size or make it bold, that will solve the problems with h1/h2